cmd/subctl/upgrade.go

```diff
@@ -165,6 +167,37 @@ func upgradeSubctl(status reporter.Interface) (string, error) {
 	return absolutePath, nil
 }
 
+func retrieveLatestReleaseTag() (string, error) {
+	// Retrieve the latest release tag from https://api.github.com/repos/submariner-io/releases/releases/latest
+	// (the duplicate "releases" portion is normal, it points to the releases of the Submariner releases project)
+	httpClient := &http.Client{Timeout: 10 * time.Second}
+
+	r, err := httpClient.Get("https://api.github.com/repos/submariner-io/releases/releases/latest")
+	if err != nil {
+		return "", fmt.Errorf("error accessing GitHub: %w", err)
```

Comment:
I think `errors.Wrap` is preferred (here and below).

```suggestion
		return "", errors.Wrap(err, "error accessing GitHub")
```

Reply:
See https://pkg.go.dev/errors

Reply:
They're equivalent tho `Wrap` also includes the stack trace. I've found `Wrap` more intuitive. But `github.com/pkg/errors` is no longer maintained and archived so there's that (even tho it's still used). Given that it's fine as is.